<launch>
	
	<node pkg="floor_plane_mapping" type="floor_plane_mapping_node" name ="floor_plane_mapping" output="screen"
	  launch-prefix="xterm -e gdb --args">
      <param name="base_frame" value="/bubbleRob" />
      <param name="world_frame" value="/world" />
      <param name="max_range" value="1.0" />
      <param name="n_samples" value="4000" />
      <param name="tolerance" value="0.05" />
      <param name="n_x" value="20" />
      <param name="n_y" value="20" />
      
      <remap from="/floor_plane_mapping/scans" to="/vrep/depthSensor"/>
  </node>
		
		
</launch>
